### DISCUSSION TITLE
Not auto generating tests

### DISCUSSION THREAD

**Discussion by GregLeung** [specmatic/specmatic#813] [Ask For Help] [answered]
Hi, is there any configuration that I can specify to stop auto generating tests, and only run the examples provided in the .yaml file? I am running the contract tests for java project

**Comment by joelrosario** [comment, marked as answer]
Hi @GregLeung, Specmatic auto-generates tests only for APIs having no examples. So if you provide even a single example for an API (a request and a 2xx response), Specmatic will use the example provided, and will not auto-generate tests for that API.

We are looking into ways to further tune the generation of requests.

Hope this helps.
